recipe/meta.yaml

```
@@ -60,7 +60,7 @@ about:
   home: https://www.gnutls.org/index.html
   license: LGPL-2.1-or-later
   license_family: LGPL
-  license_file: LICENSE
+  license_file: COPYING
```

```suggestion
  license_file: COPYING.LESSERv2
```
The core library is LGPL-2.1-or-later, which is this license file.